src/ISA/ISA.Spreadsheet/InvestigationFile/Study.fs

```diff
@@ -1,12 +1,11 @@
 namespace ISA.Spreadsheet
 
 open ISA
-open ISA.API
 open Comment
 open Remark
 open System.Collections.Generic
 
-module Study = 
+module Studies = 
```

Comment:
Should any of this be usable from js? If yes, please remember to use root modules only.